soapui/src/main/java/com/eviware/soapui/impl/rest/actions/explorer/callback/EndpointExplorerCallback.java

```diff
@@ -70,14 +70,16 @@ public RestURIParser getUrlParser(String url) {
     }
 
     public void createFromInspection(String json) {
-        Analytics.trackAction(EXPLORE_API_CLICK_SAVE_REQUEST);
         JSONObject request;
         try {
             request = new JSONObject(json);
         } catch (JSONException e) {
             SoapUI.logError(e);
             return;
         }
+        Analytics.trackAction(EXPLORE_API_CLICK_SAVE_REQUEST,
+                "HTTPMethod", HttpMethod.valueOf(extractMethod(request)).toString(),
```

Comment:
Does extractMethod(request) not return method in String format already?

Reply:
agree, changed